src/main/java/database/Database.java

```diff
@@ -122,9 +122,9 @@ public static ArrayList<Predicate> getAvailablePredicates(boolean order) {
                 predicate.setPredicateURI(result.getString("PredicateURI"));
                 predicate.setPredicateContext(new PredicateContext(result.getString("Context_Subject"), result.getString("Context_Object"), 0));
                 predicate.setWeight(result.getLong("ContextWeight"));
-                if (!predicate.getPredicateURI().contains("#")) {
+//                if (!predicate.getPredicateURI().contains("#")) {
```

Comment:
I made this change because most of the dblp predicates contains `#`, Do you think that this will affect the overall performance?